lib/tarantool_server.py

```diff
@@ -767,7 +767,10 @@ def copy_files(self):
                     if (e.errno == errno.ENOENT):
                         continue
                     raise
-        shutil.copy('.tarantoolctl', self.vardir)
+        tntctl_file = '.tarantoolctl'
+        if not os.path.exists(tntctl_file):
+            tntctl_file = os.path.join(self.TEST_RUN_DIR, '.tarantoolctl')
```

Comment:
Please, clarify it with a short comment: it is not clear that the reason to find the file in two different places in backward compatibility. Future readers of the code will not have any advice, whether it may be simplified to just copy the file from test-run.

Reply:
Comment added.